client/src/components/ShoeShowPage.js

```diff
@@ -1,43 +1,108 @@
 import React, { useState, useEffect } from "react"
+import { Redirect } from "react-router-dom"
+import ReviewList from "./ReviewList.js"
+import ReviewForm from "./ReviewForm.js"
+import translateServerErrors from "../services/translateServerErrors.js"
 
 const ShoeShowPage = props => {
-    const [shoe, setShoe] = useState({})
+
+    const { user } = props
+    const [shoe, setShoe] = useState({
+      reviews: []
+    })
+    const [shouldRedirect, setShouldRedirect] = useState(false)
+    const [errors, setErrors] = useState({})
 
     const id = props.match.params.id
     const getShoe = async () => {
         try{
             const response = await fetch(`/api/v1/shoes/${id}`)
             if (!response.ok) {
+              if (response.status===422) {
```

Comment:
for a GET request we would not expect to get back a status of 422 so this extra error handling can be removed. a 422 occurs as the response status from a failed POST request indicating that "unprocessible entity", or it was not created successfully